src/gluonts/shell/sagemaker/params.py

```diff
@@ -79,3 +78,95 @@ def encode_sagemaker_parameters(decoded_params: dict) -> dict:
     {'foo': '[1, 2, 3]', 'bar': 'hello'}
     """
     return map_dct_values(encode_sagemaker_parameter, decoded_params)
+
+
+def detrim_and_decode_sagemaker_parameters(trimmed_params: dict) -> dict:
+    """Decode a SageMaker parameters dictionary where all values are strings.
+
+    Example:
+
+    >>> detrim_sagemaker_parameters({
+    ...     '_0_foo': '[1, ', '_1_foo': '2, 3', '_2_foo': ']',
+    ...      '_0_bar': 'hell', '_1_bar': 'o'
```

Comment:
Can you change the doc-strings to be `key: value` for each line?

This ordering is confusing to me.

Reply:
Agreed!